data-processing/entities/sentences/extractor.py

```diff
@@ -50,14 +58,29 @@ def parse(self, tex_path: str, tex: str) -> Iterator[Sentence]:
         # Record the current length of the plain text so account for the extractor bug
```

Comment:
We no longer need the `char_span=True` argument in `Segmenter` since we are calculating offsets ourselves.